### PR TITLE
[JENKINS-70231] Fix tooltips on 2.379+

### PR DESCRIPTION
**Pull request by janfaracik**
Adapts to https://github.com/jenkinsci/jenkins/commit/61f6553844b671a3a0dff5454464f90f3ac466c3

### CODE REVIEW

**Comment by janfaracik** [comment]
Ping @daniel-beck

**Comment by timja** [comment]
@daniel-beck do you have an ETA for this? it's currently holding up the tippy core PR

**Review by NotMyFault — APPROVED** [review]

**Comment by daniel-beck** [comment]
Cannot test this PR on 2.380, probably due to [JENKINS-70178](https://issues.jenkins.io/browse/JENKINS-70178).

**Comment by daniel-beck** [comment]
This PR appears to break the "Add User" and "Add Group" buttons on 2.380+. If you click them, nothing happens after entering the name. No errors are logged to the console.

Additionally, the tooltips show HTML entities, indicating this doesn't work as expected:

* 2.379 + matrix-auth 3.1.5: `Overall/Administer`
* 2.380 + matrix-auth 3.1.5: `Overall&#x2F;Administer` (weekly.ci)
* 2.379 + this PR: `Overall/Administer`
* 2.380 + this PR: `Overall&#x2F;Administer` (unexpected)

**Comment by daniel-beck** [comment]
Found the problem of the PR breaking on 2.380+ and fixed it, but still unclear why the HTML entities are visible.

**Comment by daniel-beck** [comment]
> `body[data-version]` maybe? Now that it's released…

This seems to work in a quick test, pushed a commit with the proposed fix.

File: src/main/resources/hudson/security/table.js [hunk 1]
```diff
@@ -201,9 +204,16 @@ Behaviour.specify(".global-matrix-authorization-strategy-table TD.stop A.migrate
...
+ * Determine which attribute to set tooltips in. Changed in Jenkins 2.379 with Tippy and data-html-tooltip support.
+ */
 function getTooltipAttributeName() {
-  var tippySupported = window.registerTooltips !== undefined;
-  return tippySupported ? 'html-tooltip' : 'tooltip';
+  let coreVersion = document.body.getAttribute('data-version');
+  if (coreVersion === null) {
+    return 'tooltip'
+  }
+  let tippySupported = coreVersion >= '2.379';
```
Comment (timja) [inline src/main/resources/hudson/security/table.js:215]: I would think about adding a TODO, this check could possibly break with interesting version numbers
```suggestion
  // TODO remove after minimum version is 2.379 or higher
  let tippySupported = coreVersion >= '2.379';
```

Comment (daniel-beck) [reply src/main/resources/hudson/security/table.js:215]: If we get to 2.1000, yes 😁

Comment (timja) [reply src/main/resources/hudson/security/table.js:216]: heh yeah that was the only one I found it broke on I did try quite a few in the browser

Comment (daniel-beck) [reply src/main/resources/hudson/security/table.js:216]: It's an ascii-left-to-right string comparison. If we limit it to version numbers, the sort order is like in https://stats.jenkins.io/plugin-installation-trend/installations.json

**Pull request merged**